packages/app-marketplace-admin/cdk-stack.ts

```diff
@@ -0,0 +1,31 @@
+import { execSync } from 'child_process'
+import { createBaseStack, createSite } from '@reapit/ts-scripts/src/cdk'
+import { join } from 'path'
+import config from './config.json'
+
+const createStack = async () => {
+  const stack = createBaseStack({
+    namespace: 'cloud',
+    appName: 'app-marketplace-admin',
+    component: 'site',
+    accountId: '028446965111',
+    region: 'eu-west-2',
+  })
+
+  const domain = config.domain
```

Comment:
is domain the domain that the app will end up (e.g. marketplace.reapit.cloud) on or parent of that (e.g. reapit.cloud)?

Reply:
yea, did consider making it resolved by using stack name, env and cert domain